app/helpers/form_helper.rb

```diff
@@ -0,0 +1,10 @@
+module FormHelper
+  def bool_to_int bool
+    bool ? 1 : 0
+  end
+
+  def cancel_button_link record
```

Comment:
this is sponsor specific and shouldn't be part of helpers as helpers are global.